### ISSUE TITLE
prefix column needs to widen when necessary

### ISSUE THREAD

**Issue by xxiii23** [SpriteLink/NIPAP#834]
With IPv6 prefixes nested several deep, the prefix column (which appears to be a fixed 300px wide) is too narrow, resulting in formatting anomalies.

For now I'm working around this by changing .prefix_ind_pref in nipap.css to 400px wide and prefix_column width: 400px in macros.html.

**Comment by plajjan** [comment]
Right, so the whole prefix list should dynamically resize depending on screen width.

Initially when we built this thing we did try out to let the browser size it dynamically but that just failed miserably with it wrapping the column over two lines or just giving things the wrong size. I think that dynamic width has to be implemented in javascript.

Increasing one of the columns just solves the problem for you. Someone with a small screen will come along and ask for it to be decreased ;)
